snapm/manager/_manager.py

```
@@ -351,6 +364,196 @@
+
+    lockfile = f"{lockdir}/{_escape_path(name)}.lock"
+    _log_debug("Locking snapset %s (%s)", name, lockfile)
+
+    try:
+        fd = os.open(lockfile, os.O_WRONLY | os.O_CREAT | os.O_EXCL)
+    except FileExistsError as err:
+        raise busy(err) from err
+    except OSError as err:
+        raise SnapmError(
+            f"Failed to create snapshot set lockfile {lockfile}: {err}"
+        ) from err
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)